2_process/src/process_data.R

```diff
@@ -2,21 +2,15 @@
 library(dplyr)
 library(readr)
 library(stringr)
-library(sbtools)
 library(whisker)
 
-project_output_dir <- 'my_dir'
-
-dir.create(project_output_dir)
-
-
-# Get the data from ScienceBase
-mendota_file <- file.path(project_output_dir, 'model_RMSEs.csv')
-item_file_download('5d925066e4b0c4f70d0d0599', names = 'me_RMSE.csv', destinations = mendota_file, overwrite_file = TRUE)
 
 
 # Prepare the data for plotting
-eval_data <- readr::read_csv(mendota_file, col_types = 'iccd') %>%
+process_data <- function(data){ 
```

Comment:
Because this function is where plot colors and shapes are set, I would make plot color and shape arguments. These are commonly adjusted when making a figure

Reply:
Looking at your new plotting function, you could delete the colors and shapes in this function now. Or switch the plot function to use the col and pch stored in the dataframe